.env.example

```
@@ -2,9 +2,11 @@ GID=1000
 UID=1000
 
 # Django settings
+ACCOUNT_DEFAULT_HTTP_PROTOCOL="http"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)